package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "netanos",
-  "version": "1.1.5",
+  "version": "1.1.3",
```

Comment:
@maximilianmozes This is not on par with the npm version. The next one will then be 1.1.6.